server/src/test/java/org/opensearch/action/admin/cluster/health/ClusterHealthResponsesTests.java

```diff
@@ -209,7 +209,7 @@ public void testVersionCompatibleSerialization() throws IOException {
         StreamInput in_gt_7_0 = out_gt_1_0.bytes().streamInput();
```

Comment:
Should we change the text of the comments above from `discovered_master` to `discovered_cluster_manager`?

Reply:
yes; lets get the comments as we catch them in the PR review

Reply:
Hi @owaiskazi19 Thanks a lot for your review! You are so careful. 😄 I will modify those `discovered_master` names, after checking the behavior for the API response field.